src/main/java/io/vertx/pgclient/codec/decoder/message/EmptyQueryResponse.java

```diff
@@ -0,0 +1,19 @@
+package io.vertx.pgclient.codec.decoder.message;
+
+import io.vertx.pgclient.codec.Message;
+
+/**
+ *
+ * <p>
+ * Identifies the message as a response to an empty query string. (This substitutes for {@link CommandComplete})
+ *
+ * @author <a href="mailto:emad.albloushi@gmail.com">Emad Alblueshi</a>
+ */
+
+public class EmptyQueryResponse implements Message {
```

Comment:
should make a singleton with private constructor and a `public static final EmptyQueryResponse INSTANCE`